File: Deafios em C#/1094 - Experiências.cs
using System; 

//Developed by: @LucasMarcuzo
class URI1994 {
    static void Main(string[] args) { 

        int n, C=0, R=0, S=0, x, coelhos=0, ratos=0, sapos=0, total;
        float tc, tr, ts;
        char y;

        n=int.Parse(Console.ReadLine());

        int[] vet = new int[n];

        for (int i = 0; i < n; i++)
        {
            string[] array = Console.ReadLine().Split(' ');

            x=int.Parse(array[0]);
            y=char.Parse(array[1]);

            if(y == 'C')
            {
                C++;
                coelhos+=x;

            }else if(y == 'R')
            {
                R++;
                ratos+=x;

            }else if(y == 'S')
            {
                S++;
                sapos+=x;
            }
        }

        total=coelhos+ratos+sapos;


        Console.WriteLine("Total: {0} cobaias", total);
        Console.WriteLine("Total de coelhos: {0}",coelhos);
        Console.WriteLine("Total de ratos: {0}",ratos);
        Console.WriteLine("Total de sapos: {0}",sapos);

        tc=(float)coelhos/total*100;
        tr=(float)ratos/total*100;
        ts=(float)sapos/total*100;

        Console.WriteLine("Percentual de coelhos: {0} %", tc.ToString("N2"));
        Console.WriteLine("Percentual de ratos: {0} %", tr.ToString("N2"));
        Console.WriteLine("Percentual de sapos: {0} %", ts.ToString("N2"));

    }

}

File: Desafios em JavaScript/1094 - Experiências.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

var n, C=0, R=0, S=0, x, coelhos=0, ratos=0, sapos=0, total;
var tc, tr, ts;
var y;

n=parseInt(lines.shift());

for (let i = 0; i < n; i++)
{
    var ln = lines.shift().split(" ");

    x=parseInt(ln.shift());
    y=ln.shift();

    if(y === 'C' || y === 'C\r')
    {
        C++;
        coelhos+=x;

    }
    else if(y === 'R' || y === 'R\r')
    {
        R++;
        ratos+=x;

    }
    else if(y === 'S' || y === 'S\r')
    {
        S++;
        sapos+=x;
    }
}

total=coelhos+ratos+sapos;

console.log(`Total: ${total} cobaias`);
console.log(`Total de coelhos: ${coelhos}`);
console.log(`Total de ratos: ${ratos}`);
console.log(`Total de sapos: ${sapos}`);

tc=parseFloat((coelhos/total)*100);
tr=parseFloat((ratos/total)*100);
ts=parseFloat((sapos/total)*100);

console.log(`Percentual de coelhos: ${tc.toFixed(2)} %`);
console.log(`Percentual de ratos: ${tr.toFixed(2)} %`);
console.log(`Percentual de sapos: ${ts.toFixed(2)} %`);